requirements/base.txt

```diff
@@ -7,7 +7,7 @@ jmespath~=0.9.5
 PyYAML~=5.1
 cookiecutter~=1.6.0
 aws-sam-translator==1.25.0
-docker~=4.0
+docker~=4.2.0
```

Comment:
Can we comment on why we need to specify it this way? This will help us remember and communicate to any customers who might wonder why this is like this. We will need to figure out a path out of this but for now this is fine.

Reply:
Added notice for docker minor version having breaking changes.